### PR TITLE
Initial README update

### PR DESCRIPTION
**Pull request by aeschylus**

### CODE REVIEW

**Comment by jkeck** [comment]
Can I get you to kill the rdoc README in this PR too?  Just so there is no confusion over which is the right one?

File: README.md [hunk 1]
```diff
@@ -3,3 +3,16 @@
 
 
 This is the codebase for the Colligo application, showcasing digital manuscripts.
+
+## Local Development Setup
+1. Clone and run `bundle install`
+2. Run `bin/rake db:migrate RAILS_ENV=development` or an equivalent to populate the database with development data
+3. Run `bundle exec rake` to download and install supplementary service software (Jetty) and run tests
+4. Run `bundle exec rake jetty:start`
+5. Once the Jetty service has finished spinning up, run `rake colligo:fixtures` to fill in solr fixtures (as distinct for DB migration above)
```
Comment (jkeck) [inline README.md]: Everything up to here can be done initially using the `rake colligo:install` task on the first install.  Not sure if we want to call out that install task separately.

Comment (aeschylus) [inline README.md]: Ah.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,12 @@
 
 
 This is the codebase for the Colligo application, showcasing digital manuscripts.
+
+## Local Development Setup
+Run `bundle exec rake colligo:install` to install gems, jetty, migrate the DB, and start the Solr server with fixtures.
```
Comment (drh-stanford) [inline README.md]: This rake task doesn't start the Solr server for me -- I had to do that manually and run `colligo:install` twice...

```
$ bundle exec rake colligo:install
Downloading jetty
Unzipping jetty
cp /Users/drh/workspace/colligo/config/solr_configs/schema.xml /Users/drh/workspace/colligo/jetty/solr/blacklight-core/conf/
cp /Users/drh/workspace/colligo/config/solr_configs/solrconfig.xml /Users/drh/workspace/colligo/jetty/solr/blacklight-core/conf/
rake aborted!
Errno::ECONNREFUSED: Connection refused
```

in other words, this worked...

```
$ bundle exec rake colligo:install # get connection refused error
$ bundle exec rake jetty:start colligo:install
```

**Comment by jkeck** [comment]
This will all be out of date since @mejackreed's changes I think?

**Comment by mejackreed** [comment]
Closing for now, as I think Colligo has been updated where this may need to be reworked.

**Pull request closed without merging**